app/globals.css

```
@@ -77,202 +80,88 @@
-  /* Resume-specific body styles */
-  .resume-body {
-    @apply bg-resumx-bg text-resumx-text font-sans;
-    background: linear-gradient(135deg, #fbfbfb 0%, #f7f7f7 100%);
-  }
-
-  /* Dark mode for resume */
-  .dark .resume-body {
-    @apply text-gray-100;
-    background: linear-gradient(135deg, #1a1a1a 0%, #0f0f0f 100%);
+    @apply bg-background text-foreground font-serif;
+    line-height: 1.5;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)